src/meghaJJune20/Bank.java

```diff
@@ -0,0 +1,77 @@
+package meghaJJune20;
+
+public class Bank {
+	
+	int countdebit;
+	int countcredit;
+	int countbalance;
+	int userid;
+	
+	static int debittotal=0;
+	static int credittotal=0;
+	static int baltotal=0;
```

Comment:
Remove the initialization of all instance variables.

Reply:
Done as per comments